D1673/P1673.md

```diff
@@ -6706,7 +6757,7 @@ public:
   using data_handle_type = typename Accessor::data_handle_type;
   using offset_policy = /* see below */;
 
-  constexpr accessor_conjugate(Accessor a);
+  constexpr conjugate_accessor(Accessor a);
 
   constexpr reference access(data_handle_type p, size_t i) const
```

Comment:
This one is conditionally `noexcept`, but `scaled_accessor::access` is unconditionally `noexcept`.  I would favor the former.